pixart.js

```diff
@@ -1 +1,33 @@
+var brush = $('.brush')
+var button = $('#set-color')
+var input = $('#color-field')
+var body = $('body')
+// brush.css('background-color', '#1B4370')
+// $('brush').css('background-color', 'blue');
 
+var setColor = function (e) {
+  e.preventDefault()
+  brush.css('background-color', input.val())
+}
+
+button.on('click', setColor)
+
+for (i = 0; i < 8000; i++) {
+var square = $('<div class ="square"></div>')
+    body.append(square)
+    square.on('mouseover', function () {
```

Comment:
While not technically wrong, registering this event listener should be performed outside of the for loop. After the for loop completes, all of the `div.square` elements will be available in the DOM. We can then take advantage of JQuery's selector "magic" to target all `$('.square')` elements in a single step instead of once per loop:

```js
$('.square').on('mouseover', function () {
//...
})
```